Lombiq.Hosting.Azure.ApplicationInsights/Services/TelemetryFilter.cs

```diff
@@ -1,22 +1,25 @@
 ﻿using Microsoft.ApplicationInsights.Channel;
 using Microsoft.ApplicationInsights.DataContracts;
 using Microsoft.ApplicationInsights.Extensibility;
+using System;
 using System.Collections.Generic;
 
 namespace Lombiq.Hosting.Azure.ApplicationInsights.Services;
 
 public class TelemetryFilter : ITelemetryProcessor
 {
-    private readonly List<string> _errors = new()
+    private static readonly List<string> Errors = new()
```

Comment:
I tried it specifically not to run into something like this and I don't get this. Where is that coming from? Because it doesn't seem like an analyzer of ours but some other IDE config you have, or ReSharper or something.

Reply:
Yup, it was ReSharper, changed the rule now, so I won't run into this again 😄